.github/workflows/images.yaml

```diff
@@ -25,8 +25,10 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: cilium-cli
+          - name: cilium-cli-ci
```

Comment:
This way we can leverage the `target` in docker build. as well don't need to append `-ci` in various places.